diff --git a/c_src/build_deps.sh b/c_src/build_deps.sh
index 52a86e1..c4cd802 100755
--- a/c_src/build_deps.sh
+++ b/c_src/build_deps.sh
@@ -14,7 +14,6 @@ case "$1" in
 
     (test -d secp256k1 || git clone https://github.com/bitcoin/secp256k1)
 
-    (cd secp256k1 && git reset --hard 5a91bd768faaa974e00301e662fd8f2aa75a122a &&  ./autogen.sh && ./configure --enable-module-recovery && make)
-	#(cd secp256k1 &&  ./autogen.sh && ./configure --enable-module-recovery && make)
+	  (cd secp256k1 &&  ./autogen.sh && ./configure --enable-module-recovery && make)
     ;;
 esac
